app/backplane/power_module/include/c_power_module.h

```diff
@@ -0,0 +1,62 @@
+#ifndef C_SENSOR_MODULE_H
+#define C_SENSOR_MODULE_H
+
+#include "c_sensing_tenant.h"
+
+// F-Core Includes
+#include <f_core/c_project_configuration.h>
+#include <f_core/messaging/c_message_port.h>
+#include <f_core/os/c_task.h>
+#include <f_core/types.h>
+#include <f_core/net/application/c_udp_broadcast_tenant.h>
+
+
+class CPowerModule : public CProjectConfiguration {
+public:
+
+    // TODO(aaron) Break this apart based on telemetry frequency eventually
+    struct __attribute__((packed)) SensorData {
+      NTypes::NSensor::ShuntData RailBattery;
+      NTypes::NSensor::ShuntData Rail3v3;
+      NTypes::NSensor::ShuntData Rail5v0;
+    };
+
+    /**
+     * Constructor
+     */
+    CPowerModule();
+
+    /**
+     * See parent docs
+     */
+    void AddTenantsToTasks() override;
+
+    /**
+     * See parent docs
+     */
+    void AddTasksToRtos() override;
+
+    /**
+     * See parent docs
+     */
+    void SetupCallbacks() override;
+
+private:
+    static constexpr const char* ipAddrStr = "10.1.2.1";
+    static constexpr int telemetryBroadcastPort = 11000;
```

Comment:
do we want this kinda thing to be public somewhere? i assume itll not change much but could be nice to have them named in a common place for backplane mods talking to each other.

Reply:
At some point yes. I was thinking we do something autogenerated in the future, but this should suffice for now